doc/kubernetes/modules/ROOT/pages/running/timeout_tunning.adoc

```diff
@@ -166,7 +168,8 @@ Change the cache configuration as show and add the attributes (in bold) you want
 
 [source,xml,subs=+quotes]
 ----
-<distributed-cache name="sessions" owners="2" statistics="true" *remote-timeout="15000"*> <!--1-->
+<distributed-cache name="sessions" owners="1" statistics="true" *remote-timeout="15000"*> <!--1-->
+    <memory max-count="10000" />
```

Comment:
IMHO the max-count needs a call-out describing for which caches (sessions, clientSessions, offlineSessions, offlineClientSessions) and in what scenario (persistent sessions) this should be configured.

This would also apply for the for the YAML configuration above.

Reply:
agreed, let's leave it out here.